pkgs/dkg/drand.go

```diff
@@ -102,44 +101,47 @@ type LocalOwner struct {
 	VerifyFunc func(id uint64, msg, sig []byte) error
 	SignFunc   func([]byte) ([]byte, error)
 
-	Owner common.Address
-	Nonce uint64
-	done  chan struct{}
+	Owner              common.Address
+	Nonce              uint64
+	done               chan struct{}
+	InitiatorPublicKey *rsa.PublicKey
```

Comment:
this can be part of the wrapper that we have in `state` for the local owner, since the usage is only there it doesn't make sense to put it in here.

Reply:
moved to instWrapper at state